Dockerfile-dcrd

```diff
@@ -5,38 +5,60 @@
 FROM ubuntu:xenial
 
 LABEL description="Decred mainnet daemon"
```

Comment:
Thanks for the review @kargakis 
It's a good idea, I will do it, that will also reduce the number of cache layers generated by the image build.